dmsite/frontend/src/Profile/ProfileComponent.jsx

```diff
@@ -31,59 +84,45 @@ class Profile extends React.Component {
           }}
           >
-            <CardTitle className="pl-4 pt-4 pb-0">UserProfile</CardTitle>
-            <CardBody style={{ width: '100%', justifyContent: 'space-between' }}>
-              <InputGroup>
-                <InputGroupAddon addonType="prepend">
-                  <Button id="PopoverFocus" type="button">@Username</Button>
-                  <Popover trigger="focus" placement="bottom" target="PopoverFocus">
-                    <PopoverHeader>Focus Trigger</PopoverHeader>
-                    <PopoverBody>hi</PopoverBody>
-                  </Popover>
-                </InputGroupAddon>
-                <Input placeholder="UsernameMan" />
-              </InputGroup>
-              <br />
-              <InputGroup>
-                <InputGroupAddon addonType="prepend">
-                  <Button id="PopoverFocus" type="button">@Password</Button>
-                  <Popover trigger="focus" placement="bottom" target="PopoverFocus">
-                    <PopoverHeader>Focus Trigger</PopoverHeader>
-                    <PopoverBody>hi</PopoverBody>
-                  </Popover>
-                </InputGroupAddon>
-                <Input placeholder="Password123" />
-              </InputGroup>
-              <br />
-              <InputGroup>
-                <InputGroupAddon addonType="prepend">
-                  <Button id="PopoverFocus" type="button">@Email</Button>
-                  <Popover trigger="focus" placement="bottom" target="PopoverFocus">
-                    <PopoverHeader>Focus Trigger</PopoverHeader>
-                    <PopoverBody>hi</PopoverBody>
-                  </Popover>
-                </InputGroupAddon>
-                <Input placeholder="Example@ex.com" />
-              </InputGroup>
-              <br />
-              <InputGroup>
-                <InputGroupAddon addonType="prepend">
-                  <Button id="PopoverFocus" type="button">@Security</Button>
-                  <Popover trigger="focus" placement="bottom" target="PopoverFocus">
-                    <PopoverHeader>Focus Trigger</PopoverHeader>
-                    <PopoverBody>hi</PopoverBody>
-                  </Popover>
-                </InputGroupAddon>
-                <Input placeholder="Basic User" />
-              </InputGroup>
+            <CardHeader tag="h3" align="center" style={{ backgroundColor: '#303030', color: 'white' }}>User Profile and Settings</CardHeader>
+            <CardBody style={{ width: '100%', justifyContent: 'space-between', backgroundColor: '#3d3d3d', color: 'white' }} >
+              <div className= "panel panel-default">
+                <FormErrors formErrors={this.state.formErrors} />
+              </div>
+              <div className={`form-group ${this.errorClass(this.state.formErrors.username)}`}>
+                <label htmlFor="user">Username</label>
+                <Input type="username" required className="form-control" name="username"
```

Comment:
Just add a css file with styles for input since it looks like you're repeating the same style for input each time